### ISSUE TITLE
Cannot import a VM in UTM

### ISSUE THREAD

**Issue by TresPorCiento** [utmapp/UTM#4694]
UTM cannot import VMs just can create VMs.
Does not matter if the VM is from VMware or form Virtualbox the converted disk never boots. Just works if I create a VM and migrate all the content from the old one. A lot of work!

Ventura 13.0.1 iMac 24 - M1
* UTM Version:  4.0.9

It is really difficult to find out a clear guidance on how to do it. I expect it in the manuals...but I cannot find it. We do not need to be gurus just to import a simple VM, something so easy to do in VMware and VirtualBox but so difficult in UTM.

**Comment by DJZSwag** [comment]
Can you be more specific on what you mean by importing a VM?? I'm not a dev, but I can try to help.

**Comment by TresPorCiento** [comment]
Sure. I have tried with two type of VMs from VMware Fusion and from VirtualBox. Same 2 VMs. One W11 and the other a Redhat.
So I tried to use OVA or OVF generated by Fusion or VirtualBox. Import means I used Emulation (Other) option for Win11 and Redhat. And it didn't work. 
Just worked Win11 when I used a fresh W11 vhdx using the Emulation (Windows) option. Of course I had to migrate my own data to the new Win11. 
But I cannot do that with the Redhat, too much effort on setting at the highest level for my particular task. So I need to import it. 
In both intents W11 and Linux I saw with the Emulation(Other) option how converted to qcow2 the disks but never booted.

**Comment by DJZSwag** [comment]
Are you using ARM or x86 for your VMs?

**Comment by TresPorCiento** [comment]
intel 64 bits emulated on top of M1. The Win11 fresh install worked the Emulate (other) didn't. 
<img width="471" alt="image" src="https://user-images.githubusercontent.com/118592171/202862600-7ca1c4ff-3f26-4400-96b1-70cc957ad782.png">

**Comment by DJZSwag** [comment]
Are you using an external PC to use VMware and Virtualbox?

**Comment by TresPorCiento** [comment]
The two VMs Win11 and Linux were exported from a MacBook with i9 8cores. And I am trying to import them in the M1 in my iMac 24. Transferred thru AirDrop to my iMac.

**Comment by DJZSwag** [comment]
Welp, I couldn't think of any solutions to this. I am going to leave this to the devs.

**Comment by TresPorCiento** [comment]
Ok Thanks anyway for trying to help :)

**Comment by TresPorCiento** [comment]
Ok...I found it... I opened UTM gallery and did not download but opened in UTM the ubuntu mage. So UTM downloaded and installed ubuntu. I started and it worked in my M1. So..I copied System, QEMU and Display configurations from ubuntu to my redhat and it worked!
So perhaps an easy way for us that we are not experts is that for each flavor devs make an image in UTM gallery a small one. So we can download it and see params avoiding the headache of combining options. My 2 cents :-)
